bin/qcloud_mzqc_conversion_v0.1.py

```diff
@@ -0,0 +1,111 @@
+import os
+import json
+import pronto
+import datetime
+from typing import List, Dict, Set, Any, Optional, Callable 
+from collections import defaultdict
+from MZQC import MZQCFile as mzqc
+import click
+
+@click.command()
+@click.option('-input', help='<path to the json>', required=True)
+@click.option('-output', help='<path fot the resulting mzqc>', required=True)
+def convert_to_mzqc(input, output):
+    qccv = pronto.Ontology('https://raw.githubusercontent.com/HUPO-PSI/mzQC/master/cv/qc-cv.obo')
+    psims = pronto.Ontology('https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo')
+    name_indexed = {qccv[x].name: qccv[x] for x in qccv}
+
+    with open(input, "r") as f:
+        qcloud = json.load(f)
+        qcn = os.path.basename(f.name)
+
+    base_name: str = qcn
+    cmpltn: str = str(datetime.datetime.now())  # dummy for the completion time of the real run
+    chksm: str = qcloud['file']['checksum']  # is this SHA-1???
+    instrmnt: str = "thermo"
+        
+    meta: mzqc.MetaDataParameters = mzqc.MetaDataParameters(
+            inputFiles=[
+                mzqc.InputFile(name=base_name,location="file:///dev/null", 
+                            fileFormat=mzqc.CvParameter("MS", "MS:1000584", "mzML format"), 
+                            fileProperties=[
+                                mzqc.CvParameter(cvRef="MS", 
+                                    accession="MS:1000747", 
+                                    name="completion time", 
+                                    value=cmpltn
+                                ),
+                                mzqc.CvParameter(cvRef="MS", 
+                                    accession="MS:1000569", 
+                                    name="SHA-1", 
+                                    value=chksm
+                                ),
+                                mzqc.CvParameter(cvRef="MS", 
+                                    accession="MS:1000031", 
+                                    name="instrument model", 
+                                    value=instrmnt
+                                )
+                            ]
+                )
+            ], 
+            analysisSoftware=[
+                mzqc.AnalysisSoftware(cvRef="MS", accession="MS:1000752", name="TOPP software", version="2.4", uri="openms.de")
+            ]
+        )
+
+    QC_9000001 = {"QC:0000029": "Total number of PSMs", "QC:0000031": "Total number of uniquely identified peptides", "QC:0000032": "Total number of identified proteins", "QC:0000007": "MS2 spectra count"}
```

Comment:
I put them in a flexible loop-up structure as these cornerstone metrics should be in the QC cv soon.